### ISSUE TITLE
Working group call #19

### ISSUE THREAD

**Issue by whoabuddy** [stacksgov/pm#71]
## Meeting Details

- **Date/Time:** Wednesday, July 1, 2020, 1400 UTC
    - 7am PT/10am ET/2pm UTC/4pm CEST/10pm CST
    - [Time Zone Conversion](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Stacks+Governance+Working+Group+Call&iso=20200701T10&p1=179&ah=1)
- **Location:** Link will be posted in #governance working group channel [on Discord](https://community.blockstack.org/discord) shortly before the call.
- **Recording:** https://youtu.be/h7iShriQ3q0
- **Moderator:** @blocks8 
- **Secretary:** @whoabuddy 

## Weekly Updates 

**What happened over the past week? Standup!**
- What is everyone working on right now?
- Any blockers?
- What will happen over the next week?

**Stacks Foundation Update**
- Official board meeting last week
- meeting minutes?

## Action Items

**Stacks Foundation: Call for Board Nominations #65**
- nominations are still open?

**Stacks Foundation: Bylaws #63**
- were these officially voted in?

**Meeting Roles, Responsibilities, and Bounties #62**
- defines roles and actionable items for each meeting
- what would it take to enact this now?

**Survey: final tasks before completion**
- Archive survey results #59
- Following up with survey respondents #60 

**Governance landing page #45** 
- Configure github pages for resources #69
- Github Pages live! https://stacksgov.github.io/resources
- Open PR for Docsify: https://github.com/stacksgov/resources/pull/43

**Review draft of blog 1 #52**
- General blog [project board](https://github.com/stacksgov/pm/projects/2)

**Meeting conference software selection #14**
- request for review and feedback by July 1 https://github.com/stacksgov/pm/issues/14#issuecomment-645386957

**@HaroldDavis3's documents for FabAcademy and Two Row Wampum #54**
- request for review and feedback on items below by July 1
- [FabAcademy one-pager](https://paper.dropbox.com/doc/Stacks-Foundation-SIP-One-Pager--A1Lpo1cijWpN~o8hZ1eUNcFjAQ-kOJQR4MWXLn8dgMoppDdL)
- [Two Row Wampum Development Council (TRWDC) Proposal Issue](https://paper.dropbox.com/doc/Two-Row-Wampum-Development-Council-Proposal-Issue--A1PirMlQ1znFCHQH_fq9W~8RAg-O8hFria3cc4XwgRPoWd13)
- [TRWDC one-pager](https://www.canva.com/design/DAD9mfQWg7A/-62uq1tMji0o--XhnlB3-g/edit)
- [Stacks Foundation Distributed Network Proposal Issue](https://paper.dropbox.com/doc/Stacks-Foundation-Distributed-Network-Proposal-Issue--A1V9qfjBaPoGsBVKMAuglDB2Ag-yL9LFuxYigcjthtGncfhw)

## Open Discussion

- Town hall meeting or general public call #68
- New github labels for project management repository #67
- Stacks Foundation: Community Board Seat #66
- Governance working group roadmap #56
    - SIPs process proposal [proposals#1](https://github.com/stacksgov/proposals/issues/1)
    - Founding Document Review [proposals#9](https://github.com/stacksgov/proposals/issues/9)
    - Code of Conduct [proposals#11](https://github.com/stacksgov/proposals/issues/11)
- Fireside chat series #38
- Shared Stacksgov and Blockstack Resources #33
- Languages and Translations #32
- Establish community observer role for Blockstack PBC research calls #17
- @blockstackers [community moderation project](https://github.com/blockstackers/contribute-blockstack/projects/2)

**Resources: bringing notes back up to date**
- main table of calls, videos, and notes: https://github.com/stacksgov/resources/tree/master/calls
- Notes for Call 1 and Call 2 [resources#15](https://github.com/stacksgov/resources/issues/15)
- Notes for Call 13 #47
- Notes for Call 15 #57 
- Raw transcripts for [Call 14](https://github.com/stacksgov/resources/blob/master/calls/notes/2020-05-27-Meeting-0014.md) / [Call 16](https://github.com/stacksgov/resources/blob/master/calls/notes/2020-06-10-Meeting-0016.md) / [Call 17](https://github.com/stacksgov/resources/blob/master/calls/notes/2020-06-17-Meeting-0017.md)
    - how do we best handle this? create issues for each?

**Comment by lrettig** [comment]
I won't make today's call. Here's my update:

- Thank you to everyone who provided feedback on the follow up email for survey respondents (#60). I factored in the latest feedback, and posted a few more open questions to this doc as comments, which could still use some feedback. This is waiting for the blog post (#52) to be published, and for details on the town hall (#68) as we want the sign up for this to be the CTA at the end of the email.
- Took another look at the blog post draft and suggested a few small tweaks - this is looking great! I'm excited to see it go out
- I opened a PR to archive the survey results to the resources repo (https://github.com/stacksgov/resources/pull/42)
- Working with @whoabuddy to docsify our resources repo (https://github.com/stacksgov/resources/pull/43) - he can explain what this entails!
